addons/website_sale_delivery/views/website_sale_delivery_view.xml

```diff
@@ -15,11 +15,9 @@
                     </button>
                 </div>
             </xpath>
-            <xpath expr="//group[@name='general']" position="after">
-                <group name="website">
-                    <field name="website_published"/>
-                    <field name="website_description" attrs="{'invisible':[('website_published', '=', False)]}"/>
-                </group>
+            <xpath expr="//div[@class='oe_title']" position="after">
```

Comment:
its better to have a description field should be place after notebook (should display like terms and condition on invoice)